tools/PSDecode.psm1

```diff
@@ -733,6 +733,26 @@ function Remove_Unusable_Args_On_Linux
     return $Command
 }
 
+function Convert_Encoded_Command
+    {
+    param(
+        [Parameter( `
+            Mandatory=$True, `
+            Valuefrompipeline = $True)]
+        [String]$Command
+    )
+
+    $encoded_command_pattern = [regex]"(?i)-(?:encodedcommand|enc)\s+['`"]([a-zA-Z0-9+/]+=?=?)['`"]"

```

Comment:
The -EncodedCommand can also be written ```powershell /encodedco ZQBjAGgAbwAgAGIAZQBlAA==```, and every variation of letter removed from the word "EncodedCommand". You also don't need the quotes around the base64. I believe this Regex will be able to cover more cases:
https://github.com/CybercentreCanada/Multidecoder/blob/main/multidecoder/analyzers/shell.py#L14